### DISCUSSION TITLE
Github Account Disabled Without Notification - Need Assistance

### DISCUSSION THREAD

**Discussion by CyberLordX7** [community/community#147133] [Other Feature Feedback, Questions, & Ideas]
I recently discovered that my GitHub account, which I’ve been using for over 7 years, was disabled without any prior notification or explanation. I’ve sent multiple emails to support@gmail.com but haven’t received any response. What’s even more concerning is that I wasn’t informed of the reason for the account being disabled.

This account is linked to several important services, making the situation quite distressing. I feel this is very unfair and would deeply appreciate any guidance or assistance in resolving this matter.

Thank you in advance for your help.

**Comment by mecodeatlas** [comment]
Hi @CyberLordX7!

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

And just to confirm: you opened a ticket with our support team through [this link](https://support.github.com/request), correct? If not, I recommend following this route to ensure you get a response as soon as possible.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Oh, and you are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you.

Thank you!
